frontend/packages/console-app/src/components/tour/tour-context.ts

```diff
@@ -81,6 +83,43 @@ const getRequiredFlagsByTour = createSelector(
   (flags, requiredFlags) => pick(flags, requiredFlags),
 );
 
+type TourLocalStorageType = {
+  completed: boolean;
+};
+
+type TourLocalStorageData = {
+  [key: string]: TourLocalStorageType;
+};
+
+const TOUR_CONFIGMAP_KEY = `console.guidedTour`;
```

Comment:
maybe can store `console` as const here console-shared/src/constants/common.ts , can be used for all scenarios. WDYT?

Reply:
It's fine to keep these in their respective files as they are not being used anywhere else.